File: Beginner/Java/Others/ReverseBits.java
public class Solution {
    // you need treat n as an unsigned value
    public int reverseBits(int n) {
        int res = 0, bit = 0;
        while (bit != 32) {
            res <<= 1;
            res += n & 1;
            n >>>= 1;
            bit ++;
        }
        return res;
    }
}

File: Beginner/Python/Others/ReverseBits.py
class Solution:
    # @param n, an integer
    # @return an integer
    def reverseBits(self, n):
        #  zfill() 方法返回指定长度的字符串，原字符串右对齐，前面填充0
        return int(bin(n)[2:].zfill(32)[::-1], 2)
